plugins/bindplane_logs.yaml

```diff
@@ -0,0 +1,61 @@
+version: 0.1.0
+title: BindPlane OP Logs
+description: Log parser for BindPlane Observability Pipeline.
+parameters:
+  - name: log_paths
+    type: "[]string"
+    description: A list of file glob patterns that match the file paths to be read
+    default:
+      - "/var/log/bindplane/bindplane.log"
+  - name: exclude_file_log_path
+    type: "[]string"
+    description: A list of file glob patterns to exclude from reading
+    default:
+      - "/var/log/bindplane/bindplane-*.log.gz"
+  - name: start_at
+    description: At startup, where to start reading logs from the file (`beginning` or `end`)
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
```

Comment:
Can we add in the storage extension to ensure offsets are kept? See example in [solr](https://github.com/observIQ/observiq-otel-collector/blob/main/plugins/solr_logs.yaml#L23-L25). It should add the extension the [storage](https://github.com/observIQ/observiq-otel-collector/blob/main/plugins/solr_logs.yaml#L28) parameter and the plugin [parameter](https://github.com/observIQ/observiq-otel-collector/blob/main/plugins/solr_logs.yaml#L18-L21)

Reply:
Done